src/test/java/com/puppycrawl/tools/checkstyle/utils/CheckUtilTest.java

```diff
@@ -166,7 +168,7 @@ public void testCreateFullTypeOfArray() throws Exception {
         final DetailAST arrayTypeNode = getNodeFromFile(TokenTypes.VARIABLE_DEF)
                 .getNextSibling().getFirstChild().getNextSibling();
 
-        assertEquals("int[14x14]", CheckUtil.createFullType(arrayTypeNode).toString(),
+        assertEquals("int[][14x14]", CheckUtil.createFullType(arrayTypeNode).toString(),
```

Comment:
This was an incorrect test cases previously which was exposed by these changes. The actual declaration is:

```java
    protected int[] array = new int[10];

```